client/grpc-gateway/swagger.json

```diff
@@ -5630,9 +5630,21 @@
                   "description": "PageResponse is to be embedded in gRPC response messages where the corresponding\nrequest message has used PageRequest.\n\n message SomeResponse {\n         repeated Bar results = 1;\n         PageResponse page = 2;\n }"
                 },
                 "height": {
-                  "type": "string",
-                  "format": "int64",
-                  "title": "query block height"
+                  "title": "query block height",
```

Comment:
were these changes manual or automatic?

Reply:
automatic, I think someone didn't run `make proto-all` after changing the proto files